src/app/dock_widgets/process/gt_processdock.cpp

```diff
@@ -1399,6 +1401,91 @@ GtProcessDock::renameElement()
     }
 }
 
+void
+GtProcessDock::moveElements(const QList<QModelIndex>& source,
+                            const QModelIndex& target)
+{
+    if (!target.isValid() || source.isEmpty()) return;
+
+    QList<QModelIndex> mapped;
+
+    for (auto i : source)
+    {
+        if (i.isValid()) mapped.append(mapToSource(i));
+    }
+
+    if (mapped.isEmpty()) return;
+
+    QList<GtObject*> objToMove;
+
+    /// check if all elements to move have the same parent
+    GtObject* parent = nullptr;
+
+    for (auto j : mapped)
+    {
+        if (j.model() != gtDataModel) return;
```

Comment:
Maybe an assert is a better fit? There should be no reason that the model is not the same, or could it be?
```suggestion
        assert (j.model() == gtDataModel);
```